packages/cloudflare/src/cli/utils/run-wrangler.ts

```diff
@@ -82,6 +82,8 @@ export function runWrangler(options: BuildOptions, args: string[], wranglerOpts:
 				// Wrangler would load `.env.<wrangler env>` while we should load `.env.<process.env.NEXTJS_ENV>`
 				// See https://opennext.js.org/cloudflare/howtos/env-vars
 				CLOUDFLARE_LOAD_DEV_VARS_FROM_DOT_ENV: "false",
+				// We pass this variable to wrangler so that it can be aware that it is being run by us
```

Comment:
Let's discuss that more.
I am really not convinced that have code in wrangler that depends on Open Next (and vice versa) is the right thing to do as it introduce coupling and make harder to change things.

Reply:
As we discussed let's proceed with this 🙂